local_setup.md

```diff
@@ -0,0 +1,193 @@
+# Local Setup
+
+For developing you want to set up the renuo-cms and the renuo-upload locally. They must be set up once and can then be used by every application that uses one or both of those services.
+
+## Step by Step Manual
+
+### Install renuo-cms-api
+
+Clone git repository and run bin/setup:
+
+```
+git clone git@github.com:renuo/renuo-cms-api.git
+cd renuo-cms-api
+bin/setup
+```
+
+If you already cloned the project, you can just pull the newest version and migrate the database.
+
+### Install renuo-upload signing
+
+Clone git repository and run bin/setup:
+
+```
+git clone git@github.com:renuo/renuo-upload-signing.git
+cd renuo-upload-signing
+bin/setup
+```
+
+If you already cloned the project, you can just pull the newest version and migrate the database.
+
+### Set Up a Amazon S3 bucket
+
+1.  Log in to amazon
+
+1.  Navigate to S3 by clicking on the S3 icon:
```

Comment:
but its like that in the styleguide: http://www.cirosantilli.com/markdown-style-guide/#ordered